crates/astria-cli/src/commands/sequencer.rs

```diff
@@ -501,6 +479,10 @@ async fn submit_transaction(
     let sequencer_key = SigningKey::from(private_key_bytes);
 
     let from_address = Address::from_verification_key(sequencer_key.verification_key());
+    println!(
+        "sending tx from address: {}",
+        hex::encode(from_address.to_vec())
+    );
```

Comment:
```suggestion
    println!("sending tx from address: {from_address}");
```
address implements `Display`